lib/src/screens/user_profile_screen/my_profile_screen.dart

```diff
@@ -0,0 +1,232 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_device_type/flutter_device_type.dart';
+import 'package:mobile_app/src/widgets/notification_user.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+import '../../repository/user_data.dart';
+import '../../widgets/user_profile/bottom_navigation.dart';
+import '../../widgets/user_profile/user_information.dart';
+import '../../widgets/user_profile/main_card.dart';
+import '../../constants/constant_colors.dart';
+import '../../constants/constant_text.dart';
+
+class MyProfileScreen extends StatelessWidget {
+  const MyProfileScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    if (Device.get().isPhone) {
+      return Scaffold(
+        body: Container(
+          child: Stack(
+            children: [
+              Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  SizedBox(
+                    height: 49.h,
+                  ),
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.start,
+                    children: [
+                      Container(
+                        padding: EdgeInsets.only(left: 25.w),
+                        child: Text(
+                          UserProfileText.title,
+                          style: TextStyle(
+                            color: AppColor.primaryBlack,
+                            fontSize: 24,
+                          ),
+                        ),
+                      ),
+                      Container(
+                        padding: EdgeInsets.only(left: 124.w),
+                        child: Row(
+                          children: [
+                            Container(
+                              child: TextButton(
```

Comment:
we don't need setting button on mobile screen and back button on tablet.This problem has been solved on trello
![ảnh](https://user-images.githubusercontent.com/62281690/130203360-b897297d-68b3-4170-82c5-6a26dac4ce25.png)

Reply:
Ok thanks, i will fix it right now